examples/simple_project/main.tf

```diff
@@ -28,8 +28,10 @@ locals {
   subnet_03 = "${var.network_name}-subnet-03"
 }
 
+# [START customvpc_create]
```

Comment:
The region tag needs to use a valid product prefix: https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/code_snippets_101.md?cl=head

Reply:
Got it.